src/proxy-ui-api/frontend/src/global.ts

```diff
@@ -78,7 +78,9 @@ export enum Permissions {
   IMPORT_INTERNAL_SSL_CERT = 'IMPORT_INTERNAL_SSL_CERT', // Security server TLS certificate
   IMPORT_SIGN_CERT = 'IMPORT_SIGN_CERT',
   INIT_CONFIG = 'INIT_CONFIG',
-  REFRESH_WSDL = 'REFRESH_WSDL', // client > services > refresh WSDL
+  REFRESH_WSDL = 'REFRESH_WSDL', // client > services > refresh
+  REFRESH_REST = 'REFRESH_REST',
```

Comment:
Would be nice to add comment about what the permission is or where it's used. Helps to keep track on these and in the end it's easier to see if we have some unused permissions.

Reply:
Added comments